hugegraph-server/hugegraph-dist/src/assembly/static/bin/common-topling.sh

```
@@ -0,0 +1,306 @@
+        printf 'current platform is %s/%s\n' "$os_name" "$machine_arch" >&2
+        return 1
+    fi
+}
+
+function preload_toplingdb() {
+    local lib_dir="$1"
+    local dest_dir="$2"
+
+    require_topling_platform || return 1
+
+    local top="$(cd "$lib_dir"/../ && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)